src/cl65/main.c

```diff
@@ -706,6 +720,7 @@ static void Usage (void)
             "  -C name\t\t\tUse linker config file\n"
             "  -Cl\t\t\t\tMake local variables static\n"
             "  -D sym[=defn]\t\t\tDefine a preprocessor symbol\n"
+            "  -E Stop after the preprocessing stage\n"
```

Comment:
The description of `-E` must be aligned with the other descriptions (i.e., it needs four escaped TABs).